### DISCUSSION TITLE
Newsletter

### DISCUSSION THREAD

**Discussion by aleksandra-unchained** [unchainedshop/unchained#287] [Ideas]
How about we create a monthly newsletter informing about our new features & releases, marketing activity, e-commerce trends and events, as a way to connect with existing & potential clients, start discussions, and potentially get new ideas?

**Comment by fliptation** [comment]
Let's add a newsletter sign-up form this week.

**Comment by schmidsi** [comment]
YES! In my opinion, a good ol' newsletter is still the best channel to reach an audience. Other channels like Twitter, LinkedIn, website, etc. should all try to add more subscribers to the newsletter.
